timeseries_storage/timeseries_map/map.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2015 Square Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package timeseries_map
```

Comment:
`timeseries_map` is a full-featured, but inefficient, implementation of a `TimeseriesStorageAPI`. It can be used for setting up a demo which has no external dependencies.